portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -180,6 +180,39 @@ definition {
 		}
 	}
 
+	@description = "LRQA-55802, Validation of images for Sign In and Sign Out "
+	@priority = "5"
+	test ValidateSignInSignOut {
+		property portal.acceptance = "false";
+		property test.name.skip.portal.instance = "UIInfrastructureUsecase#RefreshWebContent";
+
+		SignOut.signOut();
+
+		Refresh();
+
+		takeScreenshot();
+
+		SikuliAssertElementPresent(locator1 = "Navigation#SCREENSHOTS_SIGN_IN");
+
+		Navigator.openSpecificURL(url = "http://localhost:8080/c/portal/login");
+
+		SikuliAssertElementPresent(locator1 = "Navigation#SCREENSHOTS_LOG_IN_MODAL");
+
+		SignIn.signInSpecificURL(url = "http://localhost:8080/c/portal/login");
```

Comment:
use SignIn._signIn instead. Test has already opened up the specific URL at this point.